frontend/packages/kubevirt-plugin/integration-tests/tests/models/wizard.ts

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable no-unused-vars, no-undef */
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { createItemButton, isLoaded } from '../../../../../integration-tests/views/crud.view';
+import { selectDropdownOption, click } from '../../../../console-shared/src/test-utils/utils';
+import { fillInput } from '../utils/utils';
+import { cloudInitConfig, storageResource } from '../utils/types';
+import { PAGE_LOAD_TIMEOUT_SECS } from '../utils/consts';
+import * as wizardView from '../../views/wizard.view';
+
+export class Wizard {
```

Comment:
This will probably change in future due to VM wizard changes made by @suomiy.

Reply:
Yes, I expect I'll have to refactor.